src/components/cards/SponsorCard.tsx

```diff
@@ -1,23 +1,25 @@
 import React from 'react';
-import { View, Text, Image } from 'react-native';
-import { Sponsor } from '@knighthacks/hackathon';
+import { View, Text, Image, Pressable, Linking } from 'react-native';
+import { APISponsorData } from '@knighthacks/hackathon';
 import CardStyles from '../../styles/cardStyles';
 import { useTheme, DarkTheme } from '@react-navigation/native';
 
 export interface SponsorCardProps {
-  sponsor: Sponsor;
+  sponsor: APISponsorData;
 }
 
 const tierColor = (tier: string) => {
-  switch (tier) {
+  switch (tier.toLowerCase()) {
     case 'silver':
       return '#c0c0c0';
     case 'gold':
       return '#FFD700';
     case 'diamond':
       return '#add8e6';
+    case 'partner':
+      return '#6AACC5';
     default:
-      return 'red';
+      return '#CD7F32';
```

Comment:
I would log an error that a sponsor in the db does not have a tier assigned to it because that shouldn't happen.